usaspending_api/common/tests/unit/test_spark_helpers.py

```diff
@@ -0,0 +1,16 @@
+from usaspending_api.common.helpers.spark_helpers import clean_postgres_sql_for_spark_sql
+
+
+def test_clean_postgres():
```

Comment:
Could you add either another test or add to this test where it cleans a SQL select statement? I only bring it up because we ran into that case where it was removing too many parenthesis and it would be good to prevent that in the future.